spring-cloud-gcp-docs/src/main/asciidoc/spanner.adoc

```diff
@@ -256,7 +256,11 @@ Read https://cloud.google.com/spanner/docs/schema-and-data-model#primary_keys[Sp
 
 All accessible properties on POJOs are automatically recognized as a Cloud Spanner column.
 Column naming is generated by the `PropertyNameFieldNamingStrategy` by default defined on the `SpannerMappingContext` bean.
-The `@Column` annotation optionally provides a different column name than that of the property.
+The `@Column` annotation optionally provides a different column name than that of the property and some other settings:
+- `name` is the optional name of the column
+- `generateSchemaMaxLength` specifies for `ARRAY` and `BYTES` columns the maximum length. This setting is only used when generating DDL schema statements based on domain types.
+- `nullable` specifies if the column is created as `NOT NULL`. This setting is only used when generating DDL schema statements based on domain types.
+- `columnTypeCode` is the Cloud Spanner column type you can optionally specify. If this is not specified then a compatible column type is inferred from the Java property type.
```

Comment:
maybe just name it columnType for brevity? It is clear that it should be TypeCode from parameter type